plugins/browser-plugin-webview/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import { BrowserPlugin, BrowserTracker, Payload } from '@snowplow/browser-tracker-core';
+import { hasMobileInterface, trackWebViewEvent } from '@snowplow/webview-tracker';
+import { Logger, SelfDescribingEvent, SelfDescribingJson } from '@snowplow/tracker-core';
+
+const _trackers: Record<string, BrowserTracker> = {};
+
+export function WebViewPlugin(): BrowserPlugin {
+  let LOG: Logger;
+
+  return {
+    activateBrowserPlugin: (tracker: BrowserTracker) => {
+      _trackers[tracker.id] = tracker;
+    },
+
+    logger: (logger: Logger) => {
+      LOG = logger;
+    },
+
+    filter: (payload) => {
+      if (hasMobileInterface() === true) {
+        LOG.debug(`Payload (event ID: ${payload.eid}) was filtered out and forwarded to WebView tracker.`);
+
+        let atomicProperties = {
+          eventName: payload.e as string,
+          trackerVersion: payload.tv as string,
+          useragent: payload.ua as string,
+          url: payload.url as string,
+          title: payload.page as string,
+          referrer: payload.refr as string,
+          category: payload.se_ca as string,
+          action: payload.se_ac as string,
+          label: payload.se_la as string,
+          property: payload.se_pr as string,
+          value: payload.se_va as number,
+          minXOffset: payload.pp_mix as number,
```

Comment:
The values in the payload are string and type casting `as number` won't really change the type. I think you will need to use `parseInt` to convert strings to integers and `parseFloat` to convert strings to floats.